### ISSUE TITLE
Run tectonic on vagrant

### ISSUE THREAD

**Issue by qiujian16** [coreos/tectonic-forum#120]
Hi,

We would like to try tectonic on the local laptop, but it seems that it is not possible to install tectonic by vagrant, any quick way to install tectonic on my laptop?

**Comment by robszumski** [comment]
Agreed, this is something that would make sense for Tectonic to ship/maintain.

**Comment by IvanCherepov** [comment]
Try Tectonic Sandbox(https://coreos.com/tectonic/sandbox)!

**Comment by xiaods** [comment]
it always print: 
==> c1: Tectonic is still starting, sleeping 30 seconds
==> c1: Tectonic is still starting, sleeping 30 seconds

i can't know the process, only waiting. in china mainland, the network is not stable, i don't know  when it can done.

**Comment by molon** [comment]
same with @xiaods . I have used vpn.

**Comment by Magicloud** [comment]
Agreed on @xiaods , I need more information to make sure everything goes well.
